packages/expo-local-authentication/ios/EXLocalAuthentication/EXLocalAuthentication.m

```diff
@@ -56,8 +56,14 @@ @implementation EXLocalAuthentication
   NSError *error = nil;
 
   BOOL isSupported = [context canEvaluatePolicy:LAPolicyDeviceOwnerAuthenticationWithBiometrics error:&error];
-  BOOL isEnrolled = isSupported && error == nil;
-
+  BOOL isEnrolled;
+  
+  if (@available(iOS 11.0, *)) {
+    isEnrolled = (isSupported && error == nil) || error.code == LAErrorBiometryLockout ;
+  } else {
+    isEnrolled = (isSupported && error == nil) || error.code == LAErrorTouchIDLockout ;
+  }
```

Comment:
isEnrolledAsync returns false just during the lockout mode.

![0-02-05-9012954e633269f4c82a7998706ba66d7030d3a75a37d7a3223695f4bbd538ee_1c6da1043210d3](https://user-images.githubusercontent.com/56818655/103636738-1352b780-4f53-11eb-8b4a-68cb5ae78d70.png)


As per documentation **isEnrolledAsync** returns whether the device has saved fingerprints or facial id **regardless** the lockout mode, so i expect that **isEnrolledAsync** returns true during the lockout mode or after the lockout time passed because the device has already saved biometric.